app/assets/stylesheets/booklists.css.scss

```diff
@@ -4,6 +4,49 @@ $red-color: #d9534f;
   margin-bottom: 10px;
 }
 
+.booklist-title {
+  font-weight: bold;
+  font-size: x-large;
+  text-align: center;
+}
+
+.add-books {
+  padding-left: 5px;
+}
+
+.flag {
+  height: 30px;
+  opacity: 0.15;
+  width: 30px;
+}
+
+.flag:hover {
+  cursor: pointer;
+  cursor: hand;
+}
+
+tr.flagged-admin {
+  background-color: #fbff97;
+}
+
+tr.flagged-vip {
+  background-color: #ffe7e7;
+}
+
+img.flagged {
+  opacity: 1;
+}
+
+.remove {
+  color: #ff6969;
```

Comment:
Color literals like `#ff6969` should only be used in variable declarations; they should be referred to via variable everywhere else.